assignment1/assignment.rb

```diff
@@ -0,0 +1,192 @@
+# Compute the sum of cubes for a given range a through b. Write a method called sum_of_cubes to accomplish this task. Example Given range 1 to 3 the method should return 36.
+def sum_of_cubes num_range
+    sum = 0
+    num_range.each { |arr|
+        sum += arr ** 3
+    }
+    sum
+end
+
+sum_of_cubes (1..4)
+
+# Given an Array, return the elements that are present exactly once in the array. You need to write a method called non_duplicated_values to accomplish this task. Example: Given [1,2,2,3,3,4,5], the method should return [1,4,5]
+
+def non_duplicated_values dupl_arr
+    uniq_arr = [];
+    dupl_arr.each { |val|
+        if !uniq_arr.include? val
+            uniq_arr = uniq_arr.push(val)
+        else
+            uniq_arr.delete(val)
```

Comment:
This is still not addressed, please check

Reply:
It is not a mistake

Reply:
Ok got it, this was a miss from my end. Thanks.